packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.module.scss

```diff
@@ -1,4 +1,20 @@
+// Themes
+@import './clinical-lowlight-theme/EmbeddedContentConsumer.module';
+@import './orion-fusion-theme/EmbeddedContentConsumer.module';
+
 :local {
+  .iframe-style {
+    margin-bottom: 4px;
+    margin-top: 4px;
+  }
+
+  .iframe-focus-style {
+    margin-bottom: 4px;
+    margin-top: 4px;
```

Comment:
If the margin is already part of the iframe-style, does it need to be duplicated in the focus style?

Reply:
I just tested this and, to answer your question, no, we don't need this redundant style since the margin is already set as a default. Good catch!